recipes/pairtools/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pairtools" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 172c1bc9a043f4ecaf2b464bddd7f17dcc3e0fe417936909062485ab107c23a2
+
+build:
+  number: 1
+  skip: True # [py<35]
+  entry_points:
+    - pairtools = pairtools:cli
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - numpy
```

Comment:
If you are compiling against numpy you should use pinned_compatible() in your runtime.